airflow/config_templates/config.yml

```diff
@@ -870,6 +870,16 @@ logging:
       type: string
       example: "0o664"
       default: "0o664"
+    reset_and_split_logging:
+      description:  |
+        By default Celery sends all logs into stderr.
+        This option removes default handlers and provide new ones.
+        It might be useful to send low level logs like INFO and WARNING to stdout,
```

Comment:
IMHO this doesn't sound very explicit that if it is enabled then logs with WARNING level and below are sent to stdout .... "It might be useful" a bit confusing for me

Reply:
Do you think that:

```
By default Celery sends all logs into stderr.
If this option is selected it send low level logs like INFO and WARNING to stdout,
while sending higher severity logs to stderr.
This option removes default logging handlers, and provide new ones.
```

Explains it better, or should I give some more information there?

Reply:
I see "option is enabled" used instead of "option is selected" in other places

Also I would change "This option removes default logging handlers, and provide new ones." to
"If enabled default logging handlers will be removed, and new one provided."

Reply:
Updated.